### ISSUE TITLE
HTTP Server keep alive - Netty Server unable to process TCP incoming requests

### ISSUE THREAD

**Issue by philipWennerstrom** [reactor/reactor-netty#4]
We noted that the Reactor HTTP Server Application we are developing running on Netty at IP:PORT are failing to process incoming requests. 

We see that the TCP client repeatedly attempts to send an HTTP GET request to the application; however, it does not start processing all the messages sent.

The first request results in a successful interaction however the next one fails and logs do not show any errors but the application categorically rejected to process the next incoming request and the cycle begins again.

After an in depth investigation we noted when the failure happens the client is re-route to a different peer at IP:PORT and then this will result in a next successful interaction. Clients will not receive errors, however they experience more than 60 seconds delay for the HTTP synchronus invocation and the connection is timed out or aborted. 

If the failure is not forced by a next incoming request, the server takes 60 seconds from the previous request to successfully process. Any request received in this time period will ultimately fails.

**Comment by philipWennerstrom** [comment]
We managed to solve the problem

The responsibility of closing the connection is from the client and we should just close the connection in the server if no requests were sent by a client for a certain amount of time.

Makes sense default server's behaivor keeping the connection alive after sending the response. In our case we dont want that, we want the connections being released/closed as soon as the client gets the response. 

So we included header "Connection: close" in _response hearders_  in our server function for the get endpoint.

now every request we made is being processed by the server and we get all the responses.
